WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -1,22 +1,23 @@
+import Yosemite
```

Comment:
nit: maybe we can specify the struct/enum/class required for import? (similar for other views with Yosemite import)

Reply:
Yep! Updated here: https://github.com/woocommerce/woocommerce-ios/commit/70ff262f03fbd9830a815c2af77482dbffffd91f